ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -802,7 +801,20 @@ let on_notification server (notification : Client_notification.t) :
        after receiving this notification *)
     let configuration = Configuration.update state.configuration req in
     Fiber.return { state with configuration }
-  | DidSaveTextDocument _
+  | DidSaveTextDocument { textDocument = { uri }; _ } -> (
+    let open Fiber.O in
+    let state = Server.state server in
+    match Document_store.get_opt state.store uri with
+    | None ->
+      ( Log.log ~section:"on receive DidSaveTextDocument" @@ fun () ->
+        Log.msg "saved document is not in the store" [] );
```

Comment:
Looks to me like an error?

Reply:
I wasn't confident enough to put an assert false, so logging.

Reply:
My bad, I assumed `Log` had an `err` function, which is not the case.